package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wallet-contracts",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
bumping major version here due to slight changes in bytecode from adjusted solc optimizer settings, please amend if this should be a minor version update instead

Reply:
@acrylix I think lets reserve v2 for later, how just v1.1.0 for now?